### PR TITLE
sdgPCA edits

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/sdgPrincipalComponentAnalysis.vb [hunk 1]
```diff
@@ -57,6 +57,7 @@ Public Class sdgPrincipalComponentAnalysis
         rdoScreePlot.Checked = True
         rdoBarPlot.Checked = False
         nudDim2Var.Value = 2
+        chkResiduals.Enabled = False ' this is temporary
```
Comment (dannyparsons) [inline instat/sdgPrincipalComponentAnalysis.vb]: Does it give an error?

Comment (lilyclements) [inline instat/sdgPrincipalComponentAnalysis.vb]: No, but the residuals does not have a code set for it yet so I thought if I disable it for the time being until the code is created?

Comment (dannyparsons) [inline instat/sdgPrincipalComponentAnalysis.vb]: Yes good idea.

**Comment by dannyparsons** [comment]
Did you mean to close this?

File: instat/sdgPrincipalComponentAnalysis.vb (hunk 1 above)
Comment (lilyclements) [inline instat/sdgPrincipalComponentAnalysis.vb]: Oops sorry, that was on VB I was clearing out all the pull requests and accidentally did this one. I've re-opened it. Shall I change the name to UseGraph or keep it as Used?

Comment (dannyparsons) [inline instat/sdgPrincipalComponentAnalysis.vb]: either is fine, but that's a different pull request

**Pull request merged**